nemo/collections/nlp/nm/data_layers/bert_inference_datalayer.py

```diff
@@ -0,0 +1,68 @@
+# =============================================================================
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+from nemo.collections.nlp.nm.data_layers.text_datalayer import TextDataLayer
+from nemo.core import ChannelType, NeuralType
+from nemo.utils.decorators import add_port_docs
+
+__all__ = ['BertInferDataLayer']
+
+
+class BertInferDataLayer(TextDataLayer):
+    """
+    Data layer to run infernce with BERT (get final hidden layer).
```

Comment:
Wait... what is that? Do we have now specialized, model-dependent DataLayers for... inference?